.github/workflows/unit_test.yml

```
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Unit Tests
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [ 1.14.x, 1.15.x ]
+        platform: [ ubuntu-latest, macos-latest, windows-latest ]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Test
+        run: go test ./...
```

I'm notorious for making this sort of suggestion, but:

```suggestion
        run: go test ./...

```

---

whoops missed this, shall add it in